### PR TITLE
fgeyer: Article 3

### PR DESCRIPTION
**Pull request by circumflex3105**

### CODE REVIEW

**Comment by circumflex3105** [comment]
@hendrikebbers ready for review

File: content/posts/2023-03-14-java-release-train.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+---
+outdated: false
+showInBlog: true
+title: "Every Six Months an Update: The Path to the Current Java Release Train"
+date: 2023-03-14
+author: hendrik
+excerpt: "In recent years, there have been various changes in the Java Release Train, leading to confusion and misunderstandings within the community."
```
Comment (hendrikebbers) [inline content/posts/2023-03-14-java-release-train.md:7]: adding: This post will give an overview of the changes and how you can be better prepared for the next Java releases.

File: content/posts/2023-03-14-java-release-train.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
+---
+outdated: false
+showInBlog: true
+title: "Every Six Months an Update: The Path to the Current Java Release Train"
```
Comment (hendrikebbers) [inline content/posts/2023-03-14-java-release-train.md:4]:
```suggestion
title: "Every Six Months an Update: The Path to the Java Release Train"
```

File: content/posts/2023-03-14-java-release-train.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+With the release of Java 17, further changes have been made to the release train, and I would like to use this post to describe the specific process.
+
+## The good old days
+
+Before we look at the new release train, it makes sense to take a look at the releases of older Java versions.
+The following diagram shows the release date and lifecycle of Java 6, 7, and 8.
+The lifecycle of the versions is defined by the availability of free security updates for the respective version.
+Once there are no more free updates for a version, its lifecycle ends in the diagram.
+
+![Java Lifecycle](/posts/2023-03-14-java-release-train/lifecycle.jpg)
```
Comment (hendrikebbers) [inline content/posts/2023-03-14-java-release-train.md:25]: pls change in general all images to the centered-image tag

**Comment by circumflex3105** [comment]
@hendrikebbers Review Findings implemented.

**Pull request merged**